sci-libs/gdal/gdal-3.4.2.recipe

```diff
@@ -0,0 +1,263 @@
+	2008 Shawn Gervais
+	2008 Xiuguang Zhou (ESRI)
+	2004 2007 ITC
+	2005 2007 Ari Jolma
+	2005 2007 Kevin Ruland
+	2006-2007 Waypoint Information Technology
+	2007 Adam Guernsey
+	2007 Adam Nowacki
+	2007 Andrea Aime
+	2007 Chris Schmidt
+	2007 Christian Mueller
+	2007 Geoconcept and IGN
+	2007 Jens Oberender
+	2007 Philippe P. Vachon
+	2006 Benjamin Collins
+	2006 Christopher Condit
+	2006 James Flemer
+	2006 Kevin Locke
+	2006 MapShots Inc
+	2006 Oleg Semykin
+	2006 The MITRE Corporation
+	2005 Benjamin Simon
+	2005 Daniel Wallner
+	2005 Frans van den Bergh
+	2005 Radim Blazek
+	2005 Sam Gillingham
+	2005 Vexcel Corp
+	2004 Kor de Jong
+	2004 VIZRT Inc
+	1999-2003 Stephane Villeneuve
+	2003 Applied Coherent Technology Corporation
+	2003 OPeNDAP Inc
+	1999 2001-2002 Safe Software Inc.
+	2000-2002 Avenza Systems Inc.
+	2002 i3 - information integration and imaging Fort Collin,CO
+	2002 Mark Phillips
+	2000-2001 Atlantis Scientific Inc
+	2001 Keyhole Inc.
+	2001 Simon Perkins
+	2000 Derrick J Brashear
+	1999 Intergraph Corporation
+	1999 Les Technologies SoftMap Inc
+	1998 Global Geomatics
+	1992 Michigan State University
+	PCRaster owners"
+LICENSE="MIT
+	GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://download.osgeo.org/gdal/$portVersion/gdal-$portVersion.tar.xz"
+CHECKSUM_SHA256="16baf03dfccf9e3f72bb2e15cd2d5b3f4be0437cdff8a785bceab0c7be557335"
+PATCHES="gdal-3.4.2.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion=30.0.2
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gdal$secondaryArchSuffix = $portVersion
+	lib:libgdal$secondaryArchSuffix = $libVersionCompat
+	cmd:gdal_create
+	cmd:gdal_contour
+	cmd:gdal_grid
+	cmd:gdal_rasterize
+	cmd:gdal_translate
+	cmd:gdal_viewshed
+	cmd:gdaladdo
+	cmd:gdalbuildvrt
+	cmd:gdaldem
+	cmd:gdalenhance
+	cmd:gdalinfo
+	cmd:gdallocationinfo
+	cmd:gdalmanage
+	cmd:gdalmdiminfo
+	cmd:gdalmdimtranslate
+	cmd:gdalserver
+	cmd:gdalsrsinfo
+	cmd:gdaltindex
+	cmd:gdaltransform
+	cmd:gdalwarp
+	cmd:gnmanalyse
+	cmd:gnmmanage
+	cmd:nearblack
+	cmd:ogr2ogr
+	cmd:ogrinfo
+	cmd:ogrlineref
+	cmd:ogrtindex
+	cmd:testepsg
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libarmadillo$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libgeotiff$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libhdf5$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libnetcdf$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libproj$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libwebp$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	lib:libgeos_c$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
```

Comment:
Sort alphabetically (also in BUILD_REQUIRES)

Reply:
done